### PR TITLE
Update qqmusic.rb

### PR DESCRIPTION
**Pull request by mrdoraemoncc**
Update qqmusic.rb

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by mr-ants** [comment]
Update qqmusic.rb to 7.1.2.1

File: Casks/qqmusic.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 cask "qqmusic" do
-  version :latest
-  sha256 :no_check
+  version :"7.1.2"
+  sha256 :"f3c803f3066656ea7770cbaea3daf2c3daca56b3390f849d5130e05765ed2389"
```
Comment (miccal) [inline Casks/qqmusic.rb:3]: Please remove the colons `:` from the `version` and `sha256` values.

**Comment by vitorgalvao** [comment]
In the future, please don’t untick the box that allows edits from maintainers. This could have already been fixed and merged.

**Comment by core-code** [comment]
nothing has happened here for 18 days. in case there is still interest in converting this to versioned, i've added https://github.com/Homebrew/homebrew-cask/pull/89154

**Pull request closed without merging**